mev_inspect/schemas/prices.py

```diff
@@ -13,11 +16,39 @@
 CUSDC_TOKEN_ADDRESS = "0x39aa39c021dfbae8fac545936693ac917d5e7563"
 CDAI_TOKEN_ADDRESS = "0x5d3a536e4d6dbd6114cc1ead35777bab948e3643"
 
+TOKEN_ADDRESSES = [
+    WBTC_TOKEN_ADDRESS,
+    LINK_TOKEN_ADDRESS,
+    YEARN_TOKEN_ADDRESS,
+    AAVE_TOKEN_ADDRESS,
+    UNI_TOKEN_ADDRESS,
+    USDC_TOKEN_ADDRESS,
+    DAI_TOKEN_ADDRESS,
+    REN_TOKEN_ADDRESS,
+    CUSDC_TOKEN_ADDRESS,
+    CDAI_TOKEN_ADDRESS,
+]
+
+TOKEN_NAME_BY_ADDRESS = {
```

Comment:
these names are coingecko specific

since we don't have a `coingecko.py` i think better to name like
```
COINGECKO_NAME_BY_ADDRESS
```